types/src/v17/wallet/mod.rs

```diff
@@ -38,10 +38,16 @@ pub enum AddressPurpose {
 #[derive(Copy, Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
 #[serde(rename_all = "lowercase")]
 pub enum TransactionCategory {
-    /// Transaction is a send.
+    /// Transactions sent.
     Send,
-    /// Transactions is a receive.
+    /// Non-coinbase transactions received.
     Receive,
+    /// Coinbase transactions received with more than 100 confirmations.
```

Comment:
I think we should merge this but just flagging that now there are docs in `v17` that came from a later version of Core. We have an outstanding issue #59 that describes how the re-exports include docs so version specific types have the 'wrong' docs on them.